templates/common/_base/files/ofport-request.yaml

```diff
@@ -0,0 +1,37 @@
+mode: 0755
+path: "/etc/NetworkManager/dispatcher.d/pre-up.d/ofport-request.sh"
```

Comment:
For your consideration, usually these scripts are numbered `XX-...`

Reply:
Done